source/includes/administration/_invoice_configuration.md

```diff
@@ -0,0 +1,385 @@
+### Invoice Generation Configuration
+
+Allows you to order different invoicing steps before or after taxes when generating an invoice.
+
+<!-------------------- FIND INVOICE CONFIG -------------------->
+
+#### Find the invoice configuration associated to an organization
+
+`GET /reseller/settings/invoice-config/find?organizationId=:id`
```

Comment:
If I change it, I would need to change 3 other resource as well as UI their UI clients.

Ill open a technical debt for it because I think it can be tackled as an improvment or low hanging fruit

Reply:
Sorry did not see your second comment

Reply:
https://cloudops.atlassian.net/browse/MC-18640